src/mail.lib.php

```diff
@@ -87,29 +87,34 @@ public static function sendMail($email = '', $subject = '', $message = '', $data
 
         $subject = self::getEmailSubject($subject);
 
-        /**
-         * WordPress uses a library named PHPMailer to send emails through the
-         * provided method wp_mail, unfortunately the debug information is
-         * completely removed and this makes it difficult to troubleshoots
-         * issues reported by users when the SMTP server in their sites is
-         * misconfigured. To reduce the number of tickets related with this
-         * issue we will provide an option to allow the users to choose which
-         * technique will be used to send the alerts.
-         *
-         * @see https://github.com/PHPMailer/PHPMailer
-         * @see https://developer.wordpress.org/reference/functions/wp_mail/
-         */
-        if (SucuriScanOption::isEnabled(':use_wpmail')) {
-            wp_mail($email, $subject, $message, $headers);
-        } else {
-            @mail($email, $subject, $message, implode("\r\n", $headers));
-        }
+        try {
+            /**
+             * WordPress uses a library named PHPMailer to send emails through the
+             * provided method wp_mail, unfortunately the debug information is
+             * completely removed and this makes it difficult to troubleshoots
+             * issues reported by users when the SMTP server in their sites is
+             * misconfigured. To reduce the number of tickets related with this
+             * issue we will provide an option to allow the users to choose which
+             * technique will be used to send the alerts.
+             *
+             * @see https://github.com/PHPMailer/PHPMailer
+             * @see https://developer.wordpress.org/reference/functions/wp_mail/
+             */
+            if (SucuriScanOption::isEnabled(':use_wpmail')) {
+                wp_mail($email, $subject, $message, $headers);
+            } else {
+                @mail($email, $subject, $message, implode("\r\n", $headers));
+            }
 
-        $mails_sent = (int) self::getOption(':emails_sent');
-        self::updateOption(':emails_sent', $mails_sent + 1);
-        self::updateOption(':last_email_at', time());
+            $mails_sent = (int) self::getOption(':emails_sent');
+            self::updateOption(':emails_sent', $mails_sent + 1);
+            self::updateOption(':last_email_at', time());
 
-        return true; /* assume mail delivery */
+            return true; /* assume mail delivery */
+        } catch ( Throwable $e ) {
+            SucuriScanEvent::reportErrorEvent('An error occurred sending email - please check your server mail configuration.');
```

Comment:
**(Question)** Do we want to include details from the exception in the message, any guidance from Wordpress on error messaging

Reply:
I don't think we have any error details that we can add to the error report. Also, the `wp_mail` [helper](https://github.com/WordPress/wordpress-develop/blob/6.6/src/wp-includes/pluggable.php#L174) doesn't throw